DashAI/back/models/scikit_learn/numerical_wrapper_for_text.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer
+
+from DashAI.back.core.schema_fields import BaseSchema, component_field, int_field
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+from DashAI.back.models.scikit_learn.sklearn_like_model import SklearnLikeModel
+from DashAI.back.models.text_classification_model import TextClassificationModel
+
+
+class NumericalWrapperForTextSchema(BaseSchema):
+    """
+    NumericalWrapperForText is a metamodel that allows text classification using
+    tabular classifiers and a tokenizer.
+    """
+
+    tabular_classifier: component_field(
+        description="Tabular model used as the underlying model"
+        "to generate the text classifier.",
```

Comment:
You can improve the readability of strings in several lines by using parentheses.

```suggestion
        description=(
            "Tabular model used as the underlying model"
            "to generate the text classifier."
        ),
```